winui/Scheduler/Appointments.md

```diff
@@ -812,7 +812,7 @@ The following code example shows the usage of DataTemplate.
 </scheduler:SfScheduler>
 
 N>
-* This property will only be applicable when `EnableTooltip` is set to true.
+* This property will only be applicable when [EnableToolTip](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html#Syncfusion_UI_Xaml_Scheduler_SfScheduler_EnableToolTip) is set to true.
```

Comment:
Add API link for `CellRightPadding` in days, month view and `CellBottomPadding` in timeline view as well.